test/Microsoft.AspNetCore.SignalR.Tests/HubConnectionHandlerTests.cs

```diff
@@ -2184,6 +2184,38 @@ public int Serialize(ref byte[] bytes, int offset, T value, IFormatterResolver f
             }
         }
 
+        [Fact]
+        public async Task ServerSendsCloseWithErrorWhenConnectionClosedWithPartialMessage()
+        {
+            var serviceProvider = HubConnectionHandlerTestUtils.CreateServiceProvider(services =>
+            {
+                services.AddSignalR(options => options.EnableDetailedErrors = true);
+            });
+            var connectionHandler = serviceProvider.GetService<HubConnectionHandler<SimpleHub>>();
+
+            using (var client = new TestClient())
+            {
+                var connectionHandlerTask = await client.ConnectAsync(connectionHandler).OrTimeout();
+
+                await client.Connection.Application.Output.WriteAsync(Encoding.UTF8.GetBytes(new[] { '{' })).OrTimeout();
+
+                // Close connection
+                client.Connection.Application.Output.Complete();
+
+                // Ignore message from OnConnectedAsync
+                _ = await client.ReadAsync().OrTimeout();
```

Comment:
No need for OrTimeout if Task is ignored

Reply:
Why is there any underscore.

Reply:
+1. We assign to underscore with Task because there is a warning if you don't do anything with a returned task. Ignoring other types from a method shouldn't create a warning.